week-3/Homework/mandatory/4-eligible-students.js

```diff
@@ -7,8 +7,10 @@
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
 */
 
-function eligibleStudents() {
+function eligibleStudents(arr) {
+  let newArr = arr.filter((attendance) => attendance[1] >= 8);
 
+  return newArr.map((name) => name[0]);
 }
```

Comment:
Good working solution. Try to give more meaningful names to variables so it’s easier to understand what the code does. Variables `arr` and `newArr` should have names that describe the value they store.